src/db/models/api_token.rs

```diff
@@ -0,0 +1,10 @@
+use diesel::Queryable;
+
+#[allow(dead_code)]
+#[derive(Queryable)]
+#[diesel(table = "api_tokens")]
+pub struct APIToken {
+    token: String,
+    creation_time: String,
+    expire_time: String,
```

Comment:
why are we using `String`? Why not `[u8; 32]`, `u64` and `u64` ?

Reply:
Yes, the token should be some kind of hash. For the dates, it is actually to keep coherence with the other types we are currently using (encoded as strings).

Reply:
Also for the current backend having dates as strings makes easy to make some queries:
https://sqlite.org/lang_datefunc.html